infos.json

```diff
@@ -550,5 +550,22 @@
             "key_misc_headset_devinput": true
         }
     }
```

Comment:
you need to add a coma here

Reply:
Should I make another request or you edit it yourself